packages/terra-clinical-data-grid/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Updated jest snapshots.
```

Comment:
This was already done by #879 , can you rebase with main so your PR is up to date?

Reply:
@sdadn sorry for the late response, I had some issues with wdio that I've just fixed. I did the rebase and the PR is up to date now.